Fixes #14

mugich opened the pull request:
Added jQuery plugin into the assets folder. And then used jQuery qrcode method to generate the QR code in script.js file.

mugich committed:
Fixes #14

pkriengsiri reviewed (dismissed):
1.  Have the qr code redirect the user back to our page, e.g. https://pkriengsiri.github.io/whats-in-the-box/scan/?pasteID

2.  Add bootstrap js link back in html head

mugich committed:
fix #14

mugich commented:
Added bootstrap js link back in html head.
changed the jQuery plugin by javascript qrcode library in the assets folder. 
Created site URL with pasteID and embed in a QR code
